app/controllers/nonsipp/loansmadeoroutstanding/LoansCYAController.scala

```diff
@@ -52,7 +52,8 @@ class LoansCYAController @Inject()(
   def onPageLoad(
     srn: Srn,
     index: Max5000,
-    checkOrChange: CheckOrChange
+    checkOrChange: CheckOrChange,
+    subject: IdentitySubject
```

Comment:
Needs reverting, as this doesn't need subject argument. It's always loan recipient.